### PR TITLE
abm3 - skip scenario guid creation in EMME GUI

### PR DESCRIPTION
**Pull request by mwe-sandag**
changes to properties file - introduces checkbox to SANDAG EMME toolbox
changes to master_run file - option to skip triggering write_metadata() function

NOTE - a lot of whitespaces are removed in properties.py, lines of interest have comments to flag

### CODE REVIEW

File: src/main/emme/toolbox/utilities/properties.py [hunk 1]
```diff
@@ -30,6 +30,7 @@ class PropertiesSetter(object):
 
     useLocalDrive = _m.Attribute(bool)
     skip4Ds = _m.Attribute(bool)
+    skipNewScenarioGUID = _m.Attribute(bool)
```
Comment (mwe-sandag) [inline src/main/emme/toolbox/utilities/properties.py:33]: relevant

File: src/main/emme/toolbox/utilities/properties.py [hunk 2]
```diff
@@ -187,6 +188,7 @@ def add_properties_interface(self, pb, disclosure=False):
         skip_startup_items = [
             ("useLocalDrive",           "Use the local drive during the model run"),
             ("skip4Ds",                 "Skip running 4Ds"),
+            ("skipNewScenarioGUID",     "Skip creating new scenario guid"),
```
Comment (mwe-sandag) [inline src/main/emme/toolbox/utilities/properties.py:191]: relevant

File: src/main/emme/toolbox/utilities/properties.py [hunk 3]
```diff
@@ -324,9 +326,10 @@ def load_properties(self):
 
         self.startFromIteration = props.get("RunModel.startFromIteration", 1)
         self.sample_rates = ",".join(str(x) for x in props.get("sample_rates"))
-        
+
         self.useLocalDrive = props.get("RunModel.useLocalDrive", True)
         self.skip4Ds = props.get("RunModel.skip4Ds", False)
+        self.skipNewScenarioGUID = props.get("RunModel.skipNewScenarioGUID", False)
```
Comment (mwe-sandag) [inline src/main/emme/toolbox/utilities/properties.py:332]: relevant

**Review by anneku — APPROVED** [review]
Great addition.

**Pull request merged**